conceptual/Npgsql/release-notes/5.0.md

```diff
@@ -11,10 +11,10 @@ Npgsql 5.0 is out and available at nuget.org.
 
 The major new features of 5.0 are:
 
-*TO BE FILLED*
+* `NULL` values inside arrays of value types are now properly supported via arrays of `Nullable<T>`. Previously an attempt to read or write an array of nullable value types (e. g. `Nullable<Int32>[]`) resulted in an exception. Also `NULL` values in the database array contained the default value of the corresponding value type on the .Net side making them indistiguishable from real values (e. g. `ARRAY[0,1,NULL,3]` resulted in `Int32[]{0,1,0,3}`).
 
 Many other small improvements and performance optimizations have been introduced as well - you can track progress [here](https://github.com/npgsql/npgsql/issues?utf8=%E2%9C%93&q=milestone%3A5.0.0).
 
-## Breaking changes from 5.0
+## Breaking changes from 4.1
 
-*TO BE FILLED*
\ No newline at end of file
+* For queries returning PostgreSQL arrays of value types, the values of `NpgsqlDataReader.GetValue(int ordinal)`, `NpgsqlDataReader.GetValues(object[] values)` and overloads of `NpgsqlCommand.ExecuteScalar()` will now contain an array of the corresponding nullable value type (e. g. `Nullable<Int32>[]` for `integer[]`). Also `NpgsqlDataReader.GetFieldValue<T>(int ordinal)` will now throw an exception if an array of a non-nullable value type is requested as `T` and the actual array returned from the database contains `NULL` values (e. g. `NpgsqlDataReader.GetFieldValue<int[]>(0)` will throw for `ARRAY[0,1,NULL,3])`. Use `NpgsqlDataReader.GetFieldValue<Nullable<T>>(int ordinal)` if your array of value types might contain `NULL` values.
```

Comment:
Missing a `)` to close the second `e.g.`.

Reply:
fixed (it was wrongly included in the code block)